web/src/beta/features/PluginPlayground/WidgetsList/index.tsx

```
@@ -0,0 +1,66 @@
+  )
+    return <div />;
+
+  const widgetSchema = widgetExtension.schema.groups;
+
+  if (!widgetSchema || widgetSchema.length == 0) return <div />;
+
+  const { fields } = widgetSchema[0];
+  return (
+    <Wrapper>
+      {fields.map((field) => (
+        <PropertyItem field={field} />
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)